### ISSUE TITLE
Document unstructured mesh tally capability

### ISSUE THREAD

**Issue by gonuke** [svalinn/DAGMC#47]
There may be some data from the old web site that @gonuke can scrape up.

Next step: @gonuke will search for prior documentation

**Comment by gonuke** [comment]
@makeclean - is this still relevant - do we need to add documentation for mesh tallies?

**Comment by makeclean** [comment]
Yes we do, I've just started a documentation overhaul, no pushes yet. Ill reference this issue in the PR
